Sources/Helpers/OAAppSettings.m

```diff
@@ -1854,7 +1854,8 @@ - (instancetype) init
         _settingMapLanguageTranslit = [[NSUserDefaults standardUserDefaults] objectForKey:settingMapLanguageTranslitKey] ? [[NSUserDefaults standardUserDefaults] boolForKey:settingMapLanguageTranslitKey] : NO;
 
         _settingShowMapRulet = [[NSUserDefaults standardUserDefaults] objectForKey:settingShowMapRuletKey] ? [[NSUserDefaults standardUserDefaults] boolForKey:settingShowMapRuletKey] : YES;
-        _settingAppMode = [OAProfileInteger withKey:settingAppModeKey defValue:0];
+        _dayNightMode = [OAProfileInteger withKey:settingDayNightModeKey defValue:0];
+        [_registeredPreferences setObject:_dayNightMode forKey:settingDayNightModeKey];
```

Comment:
Keys should be identical to Android. You cannot use our internal settings' keys. All keys can be found in the profile .json file inside of .osf.